evaluation/evaluate.py

```diff
@@ -341,7 +346,7 @@ def _run_inference(self):
             )
             self.df.loc[df_group.index, "predicted_answer"] = output["answers"]  # type: ignore[union-attr]
             # Store the actual compression ratio used (if the press has one)
-            self.df.loc[df_group.index, "compression_ratio"] = self.press.compression_ratio  # type: ignore[union-attr, attr-defined]
+            self.df.loc[df_group.index, "compression_ratio"] = getattr(self.press, "compression_ratio", 0.0)  # type: ignore[union-attr, attr-defined]
```

Comment:
Minor change:

Could you change to `self.press.compression_ratio if self.press is not None else 0.0`?
If, for some reason/bug, self.press doesn't have a compression_ratio (shouldn't happen in practice), the updated code will fail rather than silently reporting 0 compression.